molecule/archive-plain-debian10/molecule.yml

```diff
@@ -12,7 +12,7 @@ platforms:
       - ${CONTROLLER_HOSTGROUP:-zookeeper}
       - ${CONTROLLER_HOSTGROUP:-zookeeper}_migration
     image: geerlingguy/docker-debian10-ansible
-    dockerfile: ../Dockerfile-debian10-archive.j2
+    dockerfile: ../Dockerfile-debian10.j2
```

Comment:
Whats the reason for modifying this dockerfile ?

Reply:
There were duplicate docker files, so cleaned them up.